File: Software/clients/perl/client_demo.pl
#!/usr/bin/perl -w

use strict;
use CubeClient;

my $host = 'localhost';
my $port = 12345;

my $index = 0;
my @data = (255, 0, 255, 0, 255, 0);  # Purple, Green

my $client = CubeClient->new($host, $port);
$client->set_data($index, \@data);

File: Software/clients/python/client_demo.py

from random import randint

from cube_client import CubeClient


#HOST='10.0.1.100'
HOST='localhost'
PORT=12345

data = [randint(0, 255) for x in range(4096 * 3)]

client = CubeClient(HOST, PORT)

client.set_data(0, data)

print 'Done.'
